lib/matplotlib/axes/_base.py

```diff
@@ -134,7 +131,7 @@ def _process_plot_format(fmt):
 
     The format is absolute in the sense that if a linestyle or marker is not
     defined in *fmt*, there is no line or marker. This is expressed by
-    returning _FORMAT_UNSET for the respective quantity.
+    returning 'none' for the respective quantity.
```

Comment:
'None'?

Reply:
'None' was for a test. I'd still would like to always use lowercase internally, but have to check where it fails.

Reply:
So the reason for the failing tests is that you cannot do `MarkerStyle('none')` because the lowercase version is not defined in `markers` here:
https://github.com/matplotlib/matplotlib/blob/75d1fbc96daaed69f04b110a149b96529e4ad0df/lib/matplotlib/markers.py#L198

Linestyle accepts 'none' because it does an ad-hoc conversion:

https://github.com/matplotlib/matplotlib/blob/75d1fbc96daaed69f04b110a149b96529e4ad0df/lib/matplotlib/lines.py#L1136


I'll stick with 'None' for now. - AFAICS, the marker documentation correctly only mentions 'None' not 'none'.